### ISSUE TITLE
Render ref on tracks and paths

### ISSUE THREAD

**Issue by math1985** [gravitystorm/openstreetmap-carto#514]
Currently, we don't render the ref tags on footways, cycleways, paths, and tracks. This would be useful for some users, especially in mountaineous/hiking areas. On the other hand, rendering ref might clutter the map (see e.g. #285).

See also:
* https://trac.openstreetmap.org/ticket/2326
* https://trac.openstreetmap.org/ticket/4970

**Comment by mkoniecz** [comment]
"in Italy for instance, lots of hiking paths are numbered", "Mountains paths for hiking/cycling are rendered without ref or name." - it seems like it is ref of hiking/cycling route, not of a path itself.

**Comment by hungerburg** [comment]
The closing of this with the instruction to create relations instead misses out on incremental gathering. E.g. I come across a guidepost and see destinations and the branching paths reasonably lie on the advertised route, so I tag the ref number from the guidepost on them.

To create a route relation, I would have to hike the route. Yet, that is not why I happened over there. Currently the information that I provided is lost on OSM-Carto. Refs on tracks are rendered.  Why not on paths?
